go-controller/pkg/ovn/egressgw_test.go

```diff
@@ -2231,11 +1682,7 @@ var _ = ginkgo.Describe("OVN Egress Gateway Operations", func() {
 						Nexthops: []string{"100.64.0.4"},
 						Match:    "inport == \"rtos-node1\" && ip4.src == $" + asv4 + " && ip4.dst != 10.128.0.0/14",
 					},
-					&nbdb.LogicalRouter{
-						Name:     ovntypes.OVNClusterRouter,
-						UUID:     ovntypes.OVNClusterRouter + "-UUID",
-						Policies: []string{"2a7a61cb-fb13-4266-a3f0-9ac5c4471123 [u2596996164]"},
-					},
+					clusterRouter([]string{"2a7a61cb-fb13-4266-a3f0-9ac5c4471123 [u2596996164]"}),
```

Comment:
maybe we can also make a function to generate the following `LogicalRouterPort` with a given Network (or even set default value to "100.64.0.4/32")